include/mata/nfa.hh

```diff
@@ -561,6 +557,7 @@ public:
      * @param states_to Set of target states.
      */
     void add_trans(State state_from, Symbol symbol, const StateSet& states_to);
+    //TODO: rename all "trans" to "transition". At lest function names.
```

Comment:
I agree, it's 2022, not middle ages.

Reply:
I think of myslelf as a knight saving abreviated names in distress.

Reply:
I think one should pay some bucks for characters wasted in naming of things.